.coveragerc

```diff
@@ -0,0 +1,10 @@
+# .coveragerc to control coverage files
+[run]
+source = flamingpy
+
+
+[report]
+# Regexes for lines to exclude from consideration
+
+exclude_lines =
+    if __name__ == .__main__.:
```

Comment:
GitHub is complaining about the absence of a newline at the end of file - do we need one?